src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -44,18 +50,49 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
   const { t } = useTranslation();
   const [openConfirmDialog, setOpenConfirmDialog] = useState(false);
   const toggleOpenConfirmDialog = () => setOpenConfirmDialog(!openConfirmDialog);
-  const { setFieldValue, setFieldTouched } = useFormikContext();
+  const { values, setFieldValue, setFieldTouched } = useFormikContext<Registration>();
+  const [downloadFile, setDownloadFile] = useState(false);
+
+  const downloadFileQuery = useQuery({
+    enabled: downloadFile,
+    queryKey: ['downloadFile', values.identifier, file.identifier],
+    queryFn: async () => {
+      const downloadFileResponse = await downloadPrivateFile(values.identifier, file.identifier);
+      if (downloadFileResponse?.id) {
+        openFileInNewTab(downloadFileResponse.id);
+      }
+      setDownloadFile(false); // Ensure that a new URL is obtained every time, due to expiration
+      return downloadFileResponse;
+    },
+    meta: { errorMessage: t('feedback.error.download_file') },
+    cacheTime: 0,
+  });
 
   return (
     <TableRow data-testid={dataTestId.registrationWizard.files.fileRow}>
       <TableCell sx={{ minWidth: '13rem' }}>
         <Box sx={{ display: 'flex', gap: '0.5rem', justifyContent: 'space-between', alignItems: 'center' }}>
           <TruncatableTypography>{file.name}</TruncatableTypography>
+        </Box>
+      </TableCell>
+
+      <TableCell>
+        <Box sx={{ display: 'flex' }}>
+          {downloadFileQuery.isFetching ? (
+            <CircularProgress />
+          ) : (
+            <Tooltip title={t('registration.files_and_license.open_file')}>
+              <IconButton size="small" onClick={() => setDownloadFile(true)}>
+                <AttachFileIcon color="primary" />
```

Comment:
Muligens gjemme eller disable knappen før man har lagret publikasjonen, da den ikke har noen effekt da?

Reply:
Hmm godt poeng, det er egentlig en bug det. Snakket med Rurik om det i går, og at alle filer egentlig burde være koblet til en registrering. Det er jo ikke logisk at man må lagre før man skal kunne se fila igjen 🤔 
Ulempen med å legge til fila automatisk etter opplasting er at man kan få opp filer selv om man ikke selv aktivt har lagret, men syns ikke det er helt ulogisk heller.
Tar en prat med Rurik om vi skal fikse problemet med at vi kan ha filer uten kobling til registreringer først som sist, og at vi dermed kan basere oss på at den er der

Reply:
Vet ikke når det skal virke eller ikke da, med mindre vi legger inn en god del ekstra logikk for å sammenligne orginale Publication fra APIet og det som er lokalt i frontend. Mye herk for å glatte over en bug :( 

Så tar å legger inn ny api-metode slik at det vises feilmelding til bruker, og lager en ny task på å løse rot-årsaken, nemlig at fila ikke automatisk blir knyttet til registreringa

Reply:
Oppgave for å løse problemet: https://unit.atlassian.net/browse/NP-45721